File: esercizi_java/test/compito_python_3/Trainer.py
from Dipendenti import Dipendenti

class Trainer(Dipendenti):
    def __init__(self, cod, nome, costo, ore, specialita):
        super().__init__(cod, nome, costo)
        self.ore = ore
        self.specialita = specialita
        self.costo = costo

    def toString(self):
        return super().toString()+"\ttrainer\t"+str(self.ore)+"\t"+self.specialita+"\t-\t-\t-\t"+str(self.costo)

File: esercizi_java/NetBeansProjects/Palestra/src/palestra/Trainer.java
package palestra;


public class Trainer extends Dipendente{
    int ore;
    String specialita;
    
    public Trainer(int cod, String nome,double cost,int ore,String specialita){
        super(cod,nome,cost);
        this.ore = ore;
        this.specialita = specialita;
    }
    
    @Override
    public String toString(){
        return super.toString()+"\ttrainer\t"+ore+"\t"+specialita+"\t"+"-\t-\t-\t"+super.cost;
    }
}
